epub33/core/index.html

```diff
@@ -1340,58 +1337,55 @@ <h5>Manifest Fallbacks</h5>
+						reach, starting from a given <code>item</code>'s <code>fallback</code> attribute, represents the
+						full fallback chain for that <code>item</code>. This chain also represents the EPUB Creator's
+						preferred fallback order.</p>
 
-					<p>The <a href="#attrdef-item-fallback"><code>fallback</code> attribute</a> on the <a>manifest</a>
-						<a href="#elemdef-package-item"><code>item</code></a> element specifies the fallback for the
-						referenced Publication Resource. The <code>fallback</code> attribute's IDREF [[XML]] value MUST
-						resolve to another <code>item</code> in the <code>manifest</code>. This fallback
-							<code>item</code> MAY itself specify another fallback <code>item</code>, and so on.</p>
+					<p>There are two cases for manifest fallbacks:</p>
 
-					<p>The ordered list of all the ID references that a Reading System can reach, starting from a given
-						item's <code>fallback</code> attribute, represents the <em>fallback chain</em> for that item.
-						The order of the resources in the fallback chain represents the EPUB Creator's preferred
-						fallback order.</p>
-
-					<p>Fallback chains MUST conform to one of the following requirements, as appropriate:</p>
-
-					<ul>
-						<li>
-							<p>For Foreign Content Documents, the chain MUST contain at least one <a>EPUB Content
-									Document</a>.</p>
-						</li>
-						<li>
-							<p>For Foreign Resources for which an EPUB Creator cannot provide an intrinsic fallback, the
-								chain MUST contain at least one <a>Core Media Type Resource</a>.</p>
-						</li>
-					</ul>
-
-					<p>Fallback chains MUST NOT contain circular or self-references to <code>item</code> elements in the
-						chain.</p>
+					<dl>
+						<dt id="spine-fallbacks">Spine Fallbacks</dt>
+						<dd>
+							<p>EPUB Creators MUST specify a fallback chain for a <a>Foreign Content Document</a> to
+								ensure that Reading Systems can always render the <a>spine</a> item. In this case, the
+								chain MUST contain at least one <a>EPUB Content Document</a>.</p>
+							<p>EPUB Creators MAY provide fallbacks for EPUB Content Documents (e.g., to provide a <a
+									href="#confreq-cd-scripted-flbk">fallback for scripted content</a>).</p>
+							<p>When a fallback chain includes more than one EPUB Content Document, EPUB Creators can use
+								the <a href="#attrdef-properties"><code>properties</code> attribute</a> to differentiate
+								the purpose of each.</p>
+						</dd>
 
-					<p>EPUB Creators MAY provide fallbacks for <a>Top-Level Content Documents</a> that are EPUB Content
-						Documents (e.g., to provide <a href="#confreq-cd-scripted-flbk">fallbacks for scripted
-							content</a>).</p>
+						<dt id="content-fallbacks">Content Fallbacks</dt>
+						<dd>
+							<div class="caution">
+								<p>The purpose for keeping content fallbacks in EPUB 3 was primarily to continue to
```

Comment:
```suggestion
								<p>The original reason for introducing fallbacks in EPUB 3 was primarily to continue to
```

The term "keeping" makes sense when one is familiar with the history (I presume it came from EPUB 2), but it sounds a bit strange for people who came into this world later.  Hence my proposal for change

Reply:
But anyone who knows epub 2 knows that fallbacks didn't originate in epub 3, so it just reverses the confusion. 😄 

It also won't work logically to say we introduced fallbacks to continue to provide a means of specifying fallbacks for `img`. The precedent is missing. There was a small overlap between epub 2 and epub 3 when manifest fallbacks were the only way to handle `img`.

But to go off on a tangent, I was thinking of taking the two fallback sections currently under Foreign Resources and merging with manifest fallbacks under a "Resource Fallbacks" section. I'll ponder how to fix this note while I'm doing that.

Reply:
I've modified this in the latest commit to:

> The original purpose for content fallbacks was to specify fallback images for 

Hopefully this works both ways. If you don't know the original purpose comes from EPUB 2, you can think the original purpose was for EPUB 3 and it changes nothing.

("Original purpose" knowingly forgets all about EPUB 2's use for "out-of-line xml islands", which were forgettable, and `object` which became `bindings` in EPUB 3 and then we gave up on in place of intrinsic content in the `object`.)